pandas/tests/groupby/test_categorical.py

```diff
@@ -191,6 +191,25 @@ def test_groupby_categorical_index(self):
                 [0, 1, 2, 3], levels, ordered=True), name='cats')
         assert_frame_equal(result, expected)
 
+    def test_groupby_categorical_columns_index(self):
```

Comment:
move this to pandas/tests/groupby/test_grouper.py, which is where things like this are tested

Reply:
i can't find that file...

Reply:
sorry, test_grouping.py (move both this version and the transposed)